apps/admin/controllers/approve_talk/update.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Admin
+  module Controllers
+    module ApproveTalk
+      class Update
+        include Admin::Action
+        include Dry::Monads::Result::Mixin
+        include Import[
+          operation: 'talks.operations.approve'
+        ]
+
+        def call(params)
+          result = operation.call(params[:id])
+          if result.success?
```

Comment:
what is the difference and how you decide if you need to ask for  `.success?` or use `case when` notation?

Reply:
I'm agree, `case when` is more preferred. I'll fix it